test/core/end2end/tests/no_logging.cc

```diff
@@ -90,30 +105,23 @@ class Verifier {
     }
     auto it = allowed_logs_by_module->find(filename);
     if (it != allowed_logs_by_module->end() &&
```

Comment:
If we have this, seems like we don't need the regex?

Reply:
Regex allow lists only specific text. So if an allow listed files has a new log line, which is noisy we want that to be flagged. I am not sure if I understand your question fully though.

Reply:
I see, ok